src/xtgeo/grid3d/grid.py

```diff
@@ -1648,6 +1648,29 @@ def get_heights_above_ffl(
         """
         return _grid_etc1.get_heights_above_ffl(self, ffl=ffl, option=option)
 
+    def get_property_between_surfaces(
+        self,
+        top: xtgeo.RegularSurface,
+        base: xtgeo.RegularSurface,
+        value: int | float = 1,
+        name: str = "between_surfaces",
```

Comment:
Do we `name` as an argument here, or can we just hardcode it? Most users will set the name on the property upon export anyway I guess.

Reply:
RMS just calls it "Geometric", but I would think that many user will like their own name as an option?

Reply:
don't users need to set the name upon export to RMS anyway? Like so:
```python
prop = grd.get_property_between_surfaces(surf1, surf2, name="something")
prop.to_roxar(project, gridname, "another") 
```

Then it does not matter what name you give the property in the `get_property_between_surfaces` function?

Reply:
True for `to_roxar()`, but if you instead export to file, e,g, roff `to_file("some.roff")`, then the `.name` attributes will be applied as a name for the property in the file. So I think I keep it.